src/store/ducks/jurisdictions.ts

```diff
@@ -1,6 +1,7 @@
 import { get, keyBy, keys, pickBy, values } from 'lodash';
```

Comment:
@cKellyDesign I of the opinion that we do not need any changes made to this module.  In my view it is meant to serve just plain jurisdictions with geojson.  The generic jurisdictions module can handle the rest of jurisdictions data needs.  Thoughts?

Reply:
Agreed, if we are specifying secondary reducer modules allowing us to store different types of jurisdictions/plans/etc then we should remove the customization changes we made to the existing reducer modules.